classes/blocks/class-covers.php

```diff
@@ -58,7 +58,7 @@ class Covers extends Base_Block {
 	 */
 	public function __construct() {
 		register_block_type(
-			self::get_full_block_name(),
+			'planet4-blocks/covers-beta',
```

Comment:
If the `BLOCK_NAME` constant is changed, that affects all file names of scripts being loaded with the current logic. We only want to temporarily register the block to WP with a different name, so I changed only this usage. Because of that I had to [add a little exception in the base block](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/564/files#diff-b0e752e847c025f266c46aadf983955b5796c943773645980d725d628c73120fR140). It's not pretty, but we can revisit this afterwards and remove the workaround at that point, so that we have something that's easier to apply for all blocks.